tests/aggregation/tests.py

```diff
@@ -19,7 +19,79 @@
 
 
 class BaseAggregateTestCase(TestCase):
```

Comment:
Should we rename this test case now that it's merged with `ComplexAggregateTestCase`?